core/src/main/resources/application.yml

```diff
@@ -711,6 +711,14 @@ cb:
       sdx:
         enabled: false
 
+cluster:
+  monitoring:
+    enabled: false
+    databus:
+      processing:
+        workers: 1
```

Comment:
1 worker is enough for default? I don't know, I'm just asking.

Reply:
for every cluster one record needs to be processed in every 30 min, so 1 worker should be good enough (like UMS calls are much more frequent for clusters), but if at any point we need to scale this (as the requests will be too frequent based on the number of the clusters), workers can be increased to send requests in parallel.